### DISCUSSION TITLE
Fattura appena creata tramite API  con app privata non presente su pannello di amministrazione

### DISCUSSION THREAD

**Discussion by giovabiancia** [fattureincloud/api#56] [Q&A] [answered]
Salve, ho creato un App privata seguendo il vostro tutorial.
Sono riuscito a creare una fattura tramite l' api  da voi creata: 

**https://api-v2.fattureincloud.it/c/{{company}}/issued_documents**

Adesso dove trovo la mia fattura appena creata ? 
Perchè non è presente nel pannello di amministrazione ? 

Grazie

**Comment by valmoz** [comment, marked as answer]
Buongiorno @giovabiancia 
ho sentito il nostro supporto, e sembra che il problema sia stato già risolto con loro.

Facci sapere se serve altro,
Grazie
Mauro
